generators/app/index.js

```diff
@@ -50,6 +50,7 @@ module.exports = class extends BaseGenerator {
 
     this.interactive = this.options.interactive;
     this.installDeps = this.options.installDeps;
+    this.conflicter = this.conflicter || this.env.conflicter;
```

Comment:
@mshima I'm not sure this is working. When I `npm link` and run `yo jhipster-ionic`, I get the following error:

```
    force backend/.yo-rc.json
Error jhipster-ionic

Cannot set property 'force' of undefined
```

Reply:
@mraible done.